src/optim_problems/run_opt.jl

```diff
@@ -1,27 +1,26 @@
 """
-    run_opt(ts_data::ClustData,opt_data::OptDataCEP,opt_config::Dict{String,Any};solver::Any=CbcSolver())
+    run_opt(ts_data::ClustData,opt_data::OptDataCEP,opt_config::Dict{String,Any};optimizer::DataType)
 organizing the actual setup and run of the CEP-Problem
 """
 function run_opt(ts_data::ClustData,
                     opt_data::OptDataCEP,
-                    opt_config::Dict{String,Any};
-                    solver::Any=CbcSolver(),
-                    k_ids::Array{Int64,1}=Array{Int64,1}()
+                    opt_config::Dict{String,Any},
+                    optimizer::DataType
                     )
   #Check the consistency of the data provided
   check_opt_data_cep(opt_data)
-  cep=setup_opt_cep_basic(ts_data, opt_data, opt_config, solver; k_ids=k_ids)
+  cep=setup_opt_cep_basic(ts_data, opt_data, opt_config, optimizer, opt_config["optimizer_config"])
```

Comment:
optimizer config to hold all configurations

Reply:
What are examples that can/should go into opt_config? Potentially add to documentation.

Reply:
That's a good idea. It is optimizer specific, but I could explain that for Gurobi.